wls-briefwahl-service/src/main/resources/application-local.yml

```diff
@@ -1,2 +1,20 @@
 server:
-  port: 39146
\ No newline at end of file
+  port: 39146
+spring:
+
+  # Spring JPA
+  security:
+    oauth2:
+      resourceserver:
+        jwt:
+          jwk-set-uri: http://kubernetes.docker.internal:8100/auth/realms/${realm}/protocol/openid-connect/certs
+
+
+# Define the local keycloak realm here
+realm: wls_realm
+
+security:
+  # possible values: none, all, changing (With changing, only changing requests such as POST, PUT, DELETE are logged)
+  logging.requests: all
+  oauth2:
+    resource.user-info-uri: http://kubernetes.docker.internal:8100/auth/realms/${realm}/protocol/openid-connect/userinfo
```

Comment:
warum diese Änderungen? Im Default-Profil sind doch die Werte enthalten wie wir sie brauchen und für den Betrieb in Docker-Desktop werden die Werte entsprechend gesetzt?